Configuration/Eras/python/Era_Run2_2017_cff.py

```diff
@@ -22,8 +22,9 @@
 from Configuration.Eras.Modifier_run2_jme_2017_cff import run2_jme_2017
 from Configuration.Eras.Modifier_run2_jme_2016_cff import run2_jme_2016
 from Configuration.Eras.Modifier_strips_vfp30_2016_cff import strips_vfp30_2016
+from Configuration.Eras.Modifier_run2_tauIDformat_cff import run2_tauIDformat
 
 Run2_2017 = cms.ModifierChain(Run2_2016.copyAndExclude([run2_muon_2016, run2_HLTconditions_2016, run2_ECAL_2016, run2_egamma_2016,pixel_2016,run2_jme_2016, strips_vfp30_2016]),
                               phase1Pixel, run2_ECAL_2017, run2_HF_2017, run2_HCAL_2017, run2_HE_2017, run2_HEPlan1_2017, 
-                              trackingPhase1, run2_GEM_2017, stage2L1Trigger_2017, run2_HLTconditions_2017, run2_muon_2017,run2_egamma_2017, ctpps_2017, run2_jme_2017)
+                              trackingPhase1, run2_GEM_2017, stage2L1Trigger_2017, run2_HLTconditions_2017, run2_muon_2017,run2_egamma_2017, ctpps_2017, run2_jme_2017, run2_tauIDformat)
```

Comment:
I think that this modifier should be added to the workflow, not to the era.
IIUC, this current update will break other workflows that actually rerun tau reco.

Reply:
Ok, could you point me to the configuration of these workflows?

Reply:
I think that the step definition is here https://github.com/cms-sw/cmssw/blob/959f56649b489b3c33a15bd502bef3df4f31ea78/Configuration/PyReleaseValidation/python/relval_steps.py#L2671-L2679

Reply:
Thanks! I'll implement it as process modifier then